### PR TITLE
Script updates.

### PR DESCRIPTION
**Pull request by miretskiy**
Various script updates:
  * When fetching tpcc results, also produce per-run cpu info file across all nodes.
  * Update run tpcc script to be more interactive.
  * Make it possible to specify TPCC_EXTRA_ARGS via environment.
  * Remove ".bak" file nonsense.

### CODE REVIEW

File: run_tpcc_experiment.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#!/bin/bash
+#!/opt/homebrew/bin/bash
```
Comment (srosenberg) [inline run_tpcc_experiment.sh:1]: Do we really need a newer version of bash?

File: cmd/generate.go [hunk 2]
```diff
@@ -220,19 +224,10 @@ function copy_result_with_retry() {
...
   fi
 
-  if [ "$2" == "tpcc-results" ]
+  if [ -n "$collect_cpu_info" ]
   then
-    result_files=$(find "$target_dir" -type f -name "*.txt")
-    for result_file in $result_files
-    do
-      prev_line=$(tail -2 "$result_file")
-      if [[ "$prev_line" != *efc* ]] || [[ $(tail -1 "$result_file" | awk '{if(int($3) > 87){print "pass"}}') != "pass" ]];
```
Comment (srosenberg) [inline cmd/generate.go:229]: Thanks for cleaning up `.bak` nonsense :) We'll need to update `analyze` to optionally include "failed" runs. I can do that in my PR [1] after we merge yours.

[1] https://github.com/cockroachlabs/cloud-report/pull/119

**Review by ZhouXing19 — APPROVED** [review]

File: run_tpcc_experiment.sh [hunk 3]
```diff
@@ -24,46 +25,87 @@ while getopts 'a:d:c:' flag; do
...
+# The rest of the arguments may contain paths to the scripts to execute.
+# If not specified, attempts to find scripts in the cloud directory.
+shift $((OPTIND - 1 ))
+scriptPaths=("$@")
+
+if [[ ${#scriptPaths[@]} == 0 ]]
+then
+  case "${cloud}" in
+    aws)
+      mapfile -t scriptPaths < <( find ./report-data/$curdate/aws -name "*.sh" )
```
Comment (ZhouXing19) [inline run_tpcc_experiment.sh:44]: I think we can switch back to 
`scriptPaths=($(find ./report-data/$curdate/aws -name "*.sh" |tr '\n' ' '))`, which doesn't require a newer bash

File: run_tpcc_experiment.sh [hunk 4]
```diff
@@ -26,44 +27,95 @@ done
...
+      then
+        extra_arg="-a $warehousePerVcpu"
+      else
+        extra_arg="-A $warehousePerVcpu"
+      fi
+
+      name_extra="$warehousePerVcpu-$run"
+      echo "tmux neww -t $session_name -n $diskname-$filename-$name_extra -d -- \"CRL_USERNAME=$USER NAME_EXTRA=$name_extra TPCC_EXTRA_ARGS='-a $warehousePerVcpu'  $scriptPath -b all -w tpcc -c ./cockroach-linux-2.6.32-gnu-amd64 -d\""
+      tmux neww -t $session_name -n $diskname-$filename-$name_extra -d -- "CRL_USERNAME=$USER NAME_EXTRA=$name_extra TPCC_EXTRA_ARGS='-a $warehousePerVcpu' $scriptPath -b all -w tpcc -c ./cockroach-linux-2.6.32-gnu-amd64 -d"
+      echo "Sleeping for 10s"
```
Comment (ZhouXing19) [inline run_tpcc_experiment.sh:115]: Looks like it should be changed to "TPCC_EXTRA_ARGS=$extra_arg"

**Review by ZhouXing19 — CHANGES_REQUESTED** [review]

**Pull request merged**